packages/core-accountsjs/core-accountsjs.js

```diff
@@ -0,0 +1,205 @@
+    convertSessionIdToMongoObjectId: false,
+    idProvider,
+    dateProvider,
+  },
+);
+
+const dbManager = new DatabaseManager({
+  sessionStorage: mongoStorage,
+  userStorage: mongoStorage,
+});
+
+const accountsServerOptions = {
+  siteUrl: Meteor.absoluteUrl(),
+  // sendVerificationEmailAfterSignup: false, // It's set to false by default
+};
+
+class UnchainedAccountsPassword extends AccountsPassword {
+  async createUser(user, options) {
+    if (!user.username && !user.email) {
+      throw new AccountsJsError(
+        this.options.errors.usernameOrEmailRequired,
+        CreateUserErrors.UsernameOrEmailRequired,
+      );
+    }
+
+    if (user.username && !this.options.validateUsername(user.username)) {
+      throw new AccountsJsError(
+        this.options.errors.invalidUsername,
+        CreateUserErrors.InvalidUsername,
+      );
+    }
+
+    if (user.email && !this.options.validateEmail(user.email)) {
+      throw new AccountsJsError(
+        this.options.errors.invalidEmail,
+        CreateUserErrors.InvalidEmail,
+      );
+    }
+
+    if (user.username && (await this.db.findUserByUsername(user.username))) {
+      throw new AccountsJsError(
+        this.options.errors.usernameAlreadyExists,
+        CreateUserErrors.UsernameAlreadyExists,
+      );
+    }
+
+    if (user.email && (await this.db.findUserByEmail(user.email))) {
+      throw new AccountsJsError(
+        this.options.errors.emailAlreadyExists,
+        CreateUserErrors.EmailAlreadyExists,
+      );
+    }
+
+    if (user.password) {
+      if (!this.options.validatePassword(user.password)) {
+        throw new AccountsJsError(
+          this.options.errors.invalidPassword,
+          CreateUserErrors.InvalidPassword,
+        );
+      }
+      // eslint-disable-next-line no-param-reassign
+      user.password = await this.options.hashPassword(user.password);
+    }
+
+    const { username, email, password } = user;
+    // If user does not provide the validate function only allow some fields
+    // eslint-disable-next-line no-param-reassign
+    user = this.options.validateNewUser
+      ? await this.options.validateNewUser(user)
+      : { username, email, password };
+
+    try {
+      const userId = await this.db.createUser(user);
+
+      defer(async () => {
+        if (this.options.sendVerificationEmailAfterSignup && user.email)
+          this.sendVerificationEmail(user.email);
+
+        const userRecord = await this.db.findUserById(userId);
+        this.server
+          .getHooks()
+          .emit(ServerHooks.CreateUserSuccess, { user: userRecord, options });
+      });
+
+      return userId;
+    } catch (e) {
+      await this.server.getHooks().emit(ServerHooks.CreateUserError, user);
+      throw e;
+    }
+  }
+}
+
+export const accountsPassword = new UnchainedAccountsPassword({});
+
+class UnchainedAccountsServer extends AccountsServer {
+  destroyToken = (userId, loginToken) => {
+    this.users.update(userId, {
```

Comment:
Did you test if destroyToken works?

Reply:
On this branch, no. This was merely a basic implementation as it'd come in handy later. See https://github.com/unchainedshop/unchained/pull/241/files#diff-d1c5cccd98a611f535b5ad5f7c7786d9R130. It's not used anywhere in this branch but used to build upon in #241.